crates/bevy_reflect/src/func/info.rs

```diff
@@ -111,3 +129,174 @@ impl ReturnInfo {
         self.ownership
     }
 }
+
+/// A static accessor to compile-time type information for functions.
+///
+/// This is the equivalent of [`Typed`] for functions.
+///
+/// # Blanket Implementation
+///
+/// This trait has a blanket implementation that covers:
+/// - Functions and methods defined with the `fn` keyword
+/// - Closures that do not capture their environment
+/// - Closures that capture immutable references to their environment
+/// - Closures that capture mutable references to their environment
+/// - Closures that take ownership of captured variables
+///
+/// For each of the above cases, the function signature may only have up to 15 arguments,
+/// not including an optional receiver argument (often `&self` or `&mut self`).
+/// This optional receiver argument may be either a mutable or immutable reference to a type.
+/// If the return type is also a reference, its lifetime will be bound to the lifetime of this receiver.
+///
+/// See the [module-level documentation] for more information on valid signatures.
```

Comment:
There are some broken doc-links here :)

Reply:
I’m surprised CI didn't catch this. Good eye!